core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -42,4 +43,8 @@ class ReleasePlugin implements Plugin<Project> {
         }
     }
 
+    private String getString(Project project, String propertyName, String defaultValue) {
+        project.hasProperty(propertyName) ? project.getProperty(propertyName) : defaultValue
```

Comment:
Style and naming taken from BintrayConfiguration class. Feel free to add an issue to change project style :-)